src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/slice_scatter.cpp

```diff
@@ -83,7 +83,13 @@ class SliceScatterLayerCPUTest : public testing::WithParamInterface<SliceScatter
                                                                  in_data);
             } else {
                 // Fill the slice input2~input5 with specified data.
-                tensor = ov::Tensor{ov::element::i64, targetInputStaticShapes[i], inputValues[i - 2]};
+                auto inputValue = inputValues[i - 2];
+                if (!inputValue) {
```

Comment:
Could you please elaborate on the reason behind introducing this check? It looks like the current test configuration doesn't provide null pointers. But if it does, an input tensor is created but not initialized with valid values. So what is the expectation of using such an input?